.config-schema.json

```diff
@@ -430,6 +430,29 @@
                     "x-env-variable": "OPENFGA_CHECK_QUERY_CACHE_TTL"
                 }
             }
+        },
+        "dispatchThrottling": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "enable throttling when request's number of dispatches is high",
+                    "type": "bool",
+                    "default": "false",
+                    "x-env-variable": "OPENFGA_DISPATCH_THROTTLING_ENABLED"
+                },
+                "frequency": {
```

Comment:
nit (non-blocking): my opinion is that we shouldn't expose this as a flag yet. I think we should just choose a sensible value and if we find we need to expose it then we can do so at a later time.

Reply:
I want to give us the flexibility to adjust the level of shaping in the field without having to spin new builds. This value does not need to be set by the user and we can update the default if we find some other optimal values